parlai/core/agents.py

```diff
@@ -319,6 +319,16 @@ def load_agent_module(opt):
                 new_opt[k] = v
         new_opt['model_file'] = model_file
         model_class = get_agent_module(new_opt['model'])
+        if hasattr(model_class, 'model_version'):
+            curr_version = new_opt.get('model_version', 0)
+            if curr_version != model_class.model_version():
+                raise RuntimeError(
+                    'It looks like you are trying to load an older version of '
+                    'the selected model. Change your model argument to use '
+                    'the old version from parlai/legacy_agents: for example, '
+                    '`--model legacy:model_name:{}` or '
+                    '`-m parlai.legacy_agents.model.model_v0:ModelAgent`'
```

Comment:
did you wanna do `'parlai.legacy_agents.model.model_v{}:ModelAgent'.format(curr_version)` here as well?